src/server/game/AI/ScriptedAI/ScriptedCreature.cpp

```diff
@@ -664,6 +664,30 @@ void BossAI::UpdateAI(uint32 diff)
     DoMeleeAttackIfReady();
 }
 
+void BossAI::_DespawnAtEvade(Seconds delayToRespawn /*= 30s*/, Creature* who /*= nullptr*/)
+{
+    if (delayToRespawn < 2s)
+    {
+        LOG_ERROR("scripts.ai", "BossAI::_DespawnAtEvade: called with delay of {} seconds, defaulting to 2 (me: {}})", delayToRespawn.count(), me->GetGUID().ToString().c_str());
+        delayToRespawn = 2s;
+    }
+
+    if (!who)
+        who = me;
```

Comment:
Who? Who me kekw

Reply:
This entire function is not needed as Azerothcore implemented it in Creature instead. 
```cpp
Creature::DespawnOnEvade()
```

Reply:
Function removed ;)